ape_etherscan/client.py

```diff
@@ -92,7 +92,11 @@ def get_etherscan_api_uri(ecosystem_name: str, network_name: str):
             else "https://api-testnet.polygonscan.com/api"
```

Comment:
In line 50 of this file, we need to add this one `https://testnet.snowtrace.io/`, right?

Reply:
You added the API one, but we also need the txn URL one

Reply:
I think the transaction URL is generated automatically: https://github.com/ApeWorX/ape-etherscan/blob/main/ape_etherscan/explorer.py#L25

But I agree, https://github.com/ApeWorX/ape-etherscan/blob/main/ape_etherscan/client.py#L50 needs extra logic to refer to https://testnet.snowtrace.io/ for Fuji testnet

Reply:
See https://github.com/ApeWorX/ape-etherscan/pull/75

Reply:
thanks, this is exactly what I meant!